From 9e9b736e0735686dfca5eee2d3ea52db02ef1073 Mon Sep 17 00:00:00 2001
From: jeremyd2019 <github@jdrake.com>
Date: Fri, 20 Aug 2021 11:20:48 -0700
Subject: [PATCH] explicitly instantiate template class

Not non-template inner struct of template class.  Fixes error on clang targeting x86_64-w64-windows-gnu
---
 library/tulip-core/src/vectorgraph.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/library/tulip-core/src/vectorgraph.cpp b/library/tulip-core/src/vectorgraph.cpp
index c109d73604..cf4b12963f 100644
--- a/library/tulip-core/src/vectorgraph.cpp
+++ b/library/tulip-core/src/vectorgraph.cpp
@@ -32,8 +32,8 @@
 using namespace tlp;
 
 namespace tlp {
-template struct TLP_SCOPE VectorGraphProperty<double>::ValuesImpl;
-template struct TLP_SCOPE VectorGraphProperty<node>::ValuesImpl;
+template class TLP_SCOPE VectorGraphProperty<double>;
+template class TLP_SCOPE VectorGraphProperty<node>;
 template class TLP_SCOPE NodeProperty<node>;
 template class TLP_SCOPE NodeProperty<double>;
 template class TLP_SCOPE EdgeProperty<node>;
